templates/bare-with-kube-vip/kustomization.yaml

```diff
@@ -9,9 +9,9 @@ patches:
       value: "DirAvailable--etc-kubernetes-manifests"
     - op: add
       path: /spec/kubeadmConfigSpec/preKubeadmCommands/-
-      value: 'if [ -f "/run/kubeadm/kubeadm.yaml" ]; then mkdir -p /etc/kubernetes/manifests && ctr images pull docker.io/detiber/kube-vip:v0.3.8 && ctr run --rm --net-host docker.io/detiber/kube-vip:v0.3.8 vip /kube-vip manifest pod --arp --interface bond0 --address {{ .controlPlaneEndpoint }} --controlplane --leaderElection --metal --metalKey {{ .apiKey }} --metalProjectID ${PROJECT_ID} > /etc/kubernetes/manifests/kube-vip.yaml; fi'
+      value: 'if [ -f "/run/kubeadm/kubeadm.yaml" ]; then mkdir -p /etc/kubernetes/manifests && ctr images pull docker.io/detiber/kube-vip:v0.3.8 && ctr run --rm --net-host docker.io/detiber/kube-vip:v0.3.8 vip /kube-vip manifest pod --arp --interface bond0 --address {{ .controlPlaneEndpoint }} --controlplane --leaderElection --leaseDuration 15 --leaseRenewDuration 10 --leaseRetry 2 --metal --metalKey {{ .apiKey }} --metalProjectID ${PROJECT_ID} > /etc/kubernetes/manifests/kube-vip.yaml; fi'
     - op: add
       path: /spec/kubeadmConfigSpec/postKubeadmCommands/-
-      value: 'if [ -f "/run/kubeadm/kubeadm.yaml" ]; then kubectl --kubeconfig /etc/kubernetes/admin.conf create secret generic -n kube-system metal-cloud-config --from-literal=cloud-sa.json=''{"apiKey": "{{ .apiKey }}","projectID": "${PROJECT_ID}", "eipTag": ""}'' && kubectl --kubeconfig=/etc/kubernetes/admin.conf apply -f https://kube-vip.io/manifests/rbac.yaml && ctr run --rm --net-host docker.io/detiber/kube-vip:v0.3.8 vip /kube-vip manifest daemonset --arp --interface bond0 --address {{ .controlPlaneEndpoint }} --controlplane --leaderElection --inCluster --taint --metal --provider-config /etc/cloud-sa/cloud-sa.json | kubectl --kubeconfig=/etc/kubernetes/admin.conf apply -f - && rm /etc/kubernetes/manifests/kube-vip.yaml; fi'
+      value: 'if [ -f "/run/kubeadm/kubeadm.yaml" ]; then kubectl --kubeconfig /etc/kubernetes/admin.conf create secret generic -n kube-system metal-cloud-config --from-literal=cloud-sa.json=''{"apiKey": "{{ .apiKey }}","projectID": "${PROJECT_ID}", "eipTag": ""}'' && kubectl --kubeconfig=/etc/kubernetes/admin.conf apply -f https://kube-vip.io/manifests/rbac.yaml && ctr run --rm --net-host docker.io/detiber/kube-vip:v0.3.8 vip /kube-vip manifest daemonset --arp --interface bond0 --address {{ .controlPlaneEndpoint }} --controlplane --leaderElectionleaderElection --leaseDuration 15 --leaseRenewDuration 10 --leaseRetry 2 --inCluster --taint --metal --provider-config /etc/cloud-sa/cloud-sa.json | kubectl --kubeconfig=/etc/kubernetes/admin.conf apply -f - && rm /etc/kubernetes/manifests/kube-vip.yaml; fi'
```

Comment:
`leaderElectionleaderElection` you get one vote.

Reply:
repeat repeated on https://github.com/kubernetes-sigs/cluster-api-provider-packet/pull/269/files/0f9113924224664498419e04adab36c8ea73fd5f..74076de99e5e486370b3ebfc1bb00336251fb294#diff-8431c743cfe1f6c12bc3fba761b58b3c9915bb5a05b200f3246055d48b244f13R162